docs/source/configuration.rst

```diff
@@ -54,7 +54,7 @@ For the `pyproject.toml file`_, all valid sections start with
 
 For example, a snippet from a :code:`pyproject.toml` file:
 
-.. code-block:: toml
+.. code-block:: cfg
 
     [tool.sqlfluff.core]
     templater = "jinja"
```

Comment:
Technically this is toml, but looks more consistent to format as cfg

Reply:
Just looks a bit plainer than the rest, here's a screenshot with both:

![image](https://user-images.githubusercontent.com/10931297/150044906-0a0a7c7c-6ae3-4434-b06d-2ac28968ab7f.png)

Here's when we use `cfg` for both:

![image](https://user-images.githubusercontent.com/10931297/150044741-0f70461f-7d48-4bab-bd3e-e5709b649e58.png)

Reply:
Ahh yeah good shout let's use cfg then 👍